apps/server/src/infra/preview-generator/preview-generator.consumer.ts

```diff
@@ -18,10 +18,12 @@ export class PreviewGeneratorConsumer {
 		queue: FilesPreviewEvents.GENERATE_PREVIEW,
 	})
 	public async generatePreview(@RabbitPayload() payload: PreviewFileOptions) {
-		this.logger.debug(new PreviewActionsLoggable('PreviewGeneratorConsumer.generatePreview', payload));
+		this.logger.info(new PreviewActionsLoggable('PreviewGeneratorConsumer.generatePreview:start', payload));
 
 		const response = await this.previewGeneratorService.generatePreview(payload);
```

Comment:
try catch still missing, leading to endless loops of redelivered messages which can not get processed

Reply:
@Loki-Afro No, the CoreModule was missing in files-preview-amqp.module.ts!